jest.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
```

Comment:
Nice! I didn't know about @type

Reply:
Me neither :D It is an auto generated file